doc/man3/SSL_CTX_set_options.pod

```diff
@@ -265,6 +265,20 @@ functionality is not required. Those applications can turn this feature off by
 setting this option. This is a server-side opton only. It is ignored by
 clients.
 
+=item SSL_OP_CLEANSE_PLAINTEXT
+
+By default TLS connections keep a copy of received plaintext
+application data in a static buffer until it is overwritten by the
+next portion of data. When enabling SSL_OP_CLEANSE_PLAINTEXT
+deciphered application data is cleansed by calling OPENSSL_cleanse(3)
+after passing data to the application. Data is also cleansed when
+releasing the connection (eg. L<SSL_free(3)>).
+
+Since OpenSSL only cleanses internal buffers, the application is still
+responsible for cleansing all other buffers. Most notable this applies
+to buffers passed to functions like L<SSL_read(3)>,
+L<SSL_peek(3)|SSL_read(3)> but also like L<SSL_write(3)>.
```

Comment:
Just `L<SSL_peek(3)>` should be sufficient here (and is consistent with what we normally do). There are symlinks in place to resolve where multiple functions are defined in a single file.

Reply:
I tried with L<SSL_peek(3)> and this leads to a dead link in the html man page.
therefor i did not add it in the beginning

Reply:
at least when using links2(1)

Reply:
Hmm. Strange. But this *should* work, so if it isn't then we've got that problem throughout the man pages. Not an issue for this PR to resolve - so I think we should just have it as`L<SSL_peek(3)>` for consistency.

Reply:
Ok,

JFI: Here is what i tried:

```
./Configure --prefix=$(pwd)/tmp linux-x86_64
make install
links2 tmp/share/doc/openssl/html/man3/SSL_CTX_set_options.html
```
there are no symlinks in `tmp/share/doc/openssl/html/man3/`

Reply:
@levitte...any thoughts on that?